src/main/java/rizzler/Rizzler.java

```diff
@@ -0,0 +1,35 @@
+package rizzler;
+
+import rizzler.command.ByeCommand;
+import rizzler.command.Command;
+import rizzler.command.GreetCommand;
+
+import rizzler.task.TaskLog;
+
+import rizzler.ui.RizzlerSpeech;
+import rizzler.ui.parser.Parser;
+
+public class Rizzler {
```

Comment:
That's a very interesting name!

Reply:
> That's a very interesting name!

By the way, have you done the Pull Request. I can not find yours.